pkg/service/service.go

```diff
@@ -62,6 +64,7 @@ func NewService() *Service {
 		svcByID:         map[lb.ID]*lb.LBSVC{},
 		backendRefCount: counter.StringCounter{},
 		backendByHash:   map[string]*lb.LBBackEnd{},
+		lbmap:           &lbmap.LBBPFMap{},
```

Comment:
This should probably be passed in as a parameter to `NewService()`, as an interface rather than the concrete type?

Reply:
Hmm, then we would need to pass the `&lbmap.LBBPFMap{}` in daemon.go, which I would like to avoid doing, as ideally daemon should not know anything about `lbmap`.

Alternatively, we could introduce a new constructor (`NewServiceWithLBMap(lbmap LBMap)`). But as long as we are testing `Service` from the same package, IMHO it's fine to modify its private field.